archstrike-testing/python2-rekall/PKGBUILD

```diff
@@ -0,0 +1,47 @@
+# Maintainer: ArchStrike <team@archstrike.org>
+# Contributor: strayArch <j.a.stronz@gmail.com>
+
+buildarch=1
```

Comment:
adjust buildarch if package not "any"

Reply:
should be okay --- python based. A bit bulky to test build on my hardware right now.